paasta_tools/check_marathon_services_replication.py

```diff
@@ -77,14 +77,14 @@ def filter_healthy_marathon_instances_for_short_app_id(all_tasks, app_id):
     tasks_for_app = [
         task for task in all_tasks if task.app_id.startswith("/%s" % app_id)
     ]
-    one_minute_ago = datetime.now() - timedelta(minutes=1)
+    one_minute_ago = datetime.now(timezone.utc) - timedelta(minutes=1)
```

Comment:
You could do utcnow if you want

Reply:
nope, utcnow returns a naive datetime. The [datetime docs](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow) specifically recommend this construction instead:

> *Warning*: Because naive `datetime` objects are treated by many `datetime` methods as local times, it is preferred to use aware datetimes to represent times in UTC. As such, the recommended way to create an object representing the current time in UTC by calling `datetime.now(timezone.utc)`.

Reply:
Haha. Jeeze :(